indico/core/db/sqlalchemy/descriptions.py

```diff
@@ -38,12 +56,24 @@ def _description(cls):
             default=''
         )
 
+    @declared_attr
+    def render_mode(cls):
+        # Only add the column if there's a choice
+        # between several alternatives
+        if len(cls.possible_render_modes) > 1:
+            return db.Column(
+                PyIntEnum(RenderMode),
+                default=cls.default_render_mode,
+                nullable=False
+            )
+        else:
+            return None
```

Comment:
Ah cool. Wasn't aware of that. And the docs are not extremely clear:

http://docs.sqlalchemy.org/en/latest/orm/extensions/declarative/api.html#sqlalchemy.ext.declarative.declared_attr